### PR TITLE
SCMS Blog Markdown Formatting

### PR DESCRIPTION
**Pull request by klumb**

### CODE REVIEW

**Comment by klumb** [comment]
@samanthavenialogan, please take a look at the markdown formating and links I have added.

File: Community/News/SCMS-is-now-a-chaoss-metric.md [hunk 1]
```diff
@@ -1,54 +1,67 @@
...
+Then on May 4th, we were told that we had been accepted alongside side software developer [Ria Gupta](https://medium.com/@guptaria/my-journey-of-gsoc20-begins-268ff97c2954) for the Google Summer of Code, to write the SCMS _directly_ into CHAOSS community analytics system -- [GrimoireLab](https://chaoss.github.io/grimoirelab/) -- as a metric!
+
+![GrimoireLab](https://www.sociallyconstructed.online/uploads/2/5/6/3/25632458/grimoirelab-logo_orig.jpg)
+
+_**In this blog**_ we'll explain why we’ve been added to GrimoireLab, why we think it matters for you, and most importantly what the next steps are in this exciting new process, so you can implement the system, for free, starting today!
+
+## WHY THE SCMS WAS ADDED,​ AND WHERE IT WILL TAKE YOU
 
-In this blog we'll explain why we’ve been added to GrimoireLab, why we think it matters for you, and most importantly what the next steps are in this exciting new process, so you can implement the system, for free, starting today!
-WHY THE SCMS WAS ADDED,
```
Comment (samanthavenialogan) [inline Community/News/SCMS-is-now-a-chaoss-metric.md:6]: Not sure why this is showing up as all caps, it should be made normal.

File: Community/News/SCMS-is-now-a-chaoss-metric.md [hunk 2]
```diff
@@ -1,54 +1,67 @@
...
+
+This past January, our friends over at [CHAOSS](https://chaoss.github.io/grimoirelab/) announced a new release of health metrics, and nestled in those new metric updates was a type of metric that has not yet been featured before in CHAOSS... [Our Social Currency Metrics System (SCMS)!](https://chaoss.community/metric-social-currency-metric-system/)
+
+Then on May 4th, we were told that we had been accepted alongside side software developer [Ria Gupta](https://medium.com/@guptaria/my-journey-of-gsoc20-begins-268ff97c2954) for the Google Summer of Code, to write the SCMS _directly_ into CHAOSS community analytics system -- [GrimoireLab](https://chaoss.github.io/grimoirelab/) -- as a metric!
+
+![GrimoireLab](https://www.sociallyconstructed.online/uploads/2/5/6/3/25632458/grimoirelab-logo_orig.jpg)
+
+_**In this blog**_ we'll explain why we’ve been added to GrimoireLab, why we think it matters for you, and most importantly what the next steps are in this exciting new process, so you can implement the system, for free, starting today!
+
+## WHY THE SCMS WAS ADDED,​ AND WHERE IT WILL TAKE YOU
```
Comment (samanthavenialogan) [inline Community/News/SCMS-is-now-a-chaoss-metric.md:13]: This is a heder and it seems to be showing as such down below so it should be normal in the paragraph above.

**Comment by samanthavenialogan** [comment]
Hey Kevin,

This seems fine :)
Samantha Venia Logan
Co-Founder of SociallyConstructed.Online
307-274-5516  | samantha@SociallyConstructed.Online
6715 Autumn Ridge Dr. Unit 2, Fort Collins CO., 80525

Call me for a free marketing consultation! 307-274-5516

**Pull request merged**